doc/pages/architecture/065-groups.md

```diff
@@ -0,0 +1,16 @@
+# Groups
+
+Astarte supports creating groups of devices in a realm.
+
+In this release, groups are mainly used to provide access control, combining Astarte's [path based
+authorization](070-auth.html#authorization) with the fact that devices can be queried with a group
+URL. This makes it possible to emit tokens allowing a user to operate only on devices that belong to
+a specific group.
+
+Groups can be managed using [`astartectl`](https://github.com/astarte-platform/astartectl) or using
+[AppEngine API](/api/index.html?urls.primaryName=AppEngine API). See the [Managing Groups
+page](065-managing-groups) in the User Guide for some usage examples.
+
+Keep in mind that since groups are a propery of a device, a group is existing as long as there's
```

Comment:
*property.

Moreover, invert cause and consequence. The user cares about the fact that empty groups do not exist - then, you give details about the why is that.